### PR TITLE
Revise logic for checks and remove redundant checks

### PR DESCRIPTION
**Pull request by shloka-bhalgat-unskript**
## Description
[EN-5297](https://unskript.atlassian.net/browse/EN-5297)

### Testing
- K8s get cluster health with core services
Screenshot 2024-02-20 at 8 01 44 PM

- K8s get cluster health without core services
Screenshot 2024-02-20 at 8 01 50 PM

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository. 

[EN-5297]: https://unskript.atlassian.net/browse/EN-5297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Kubernetes/legos/k8s_get_cluster_health/k8s_get_cluster_health.py [hunk 1]
```diff
@@ -2,85 +2,130 @@
...
+            print(f"Name: {issue['name']}")
+            print(f"Namespace: {issue.get('namespace', 'N/A')}")
+            print(f"Issue: {issue['issue']}")
+            print("-" * 40)
+
+def execute_kubectl_command(handle, command: str):
+    response = handle.run_native_cmd(command)
+    if response.stderr:
+        print(f"Warning: {response.stderr}")
+        if "not found" in response.stderr:
```
Comment (shloka-bhalgat-unskript) [inline Kubernetes/legos/k8s_get_cluster_health/k8s_get_cluster_health.py:39]: Convert response.stderr to .lower()

File: Kubernetes/legos/k8s_get_cluster_health/k8s_get_cluster_health.py [hunk 2]
```diff
@@ -2,85 +2,130 @@
...
+            return None  # Service not found in the given namespace, skip this service
+    if not response or not response.stdout:
```
Comment (shloka-bhalgat-unskript) [inline Kubernetes/legos/k8s_get_cluster_health/k8s_get_cluster_health.py:41]:
```suggestion
    if response:
        if response.stdout:
            return response.stdout.strip()
        else:
             print(f"No output for command: {command}")
             return None
```

**Review by jayasimha-raghavan-unskript — APPROVED** [review]
Please take care of the comments we discussed.

**Review by jayasimha-raghavan-unskript — APPROVED** [review]

**Comment by abhishek-unskript** [comment]
For the last one (health) it appears that the requirement was simply to skip certain failures (non-core services). But the code changes seem very extensive for that implementation.

**Pull request merged**